src/Badge.tsx

```diff
@@ -19,17 +20,23 @@ const propTypes = {
 
   /**
    * The visual style of the badge
+   * A convenience prop for adding `bg-*` utility classes since they are so commonly used here.
```

Comment:
You can delete this comment.  The bg is a core prop required for background coloring.